lib/toggle.js

```diff
@@ -62,23 +63,32 @@ const ToggleBody = styled.label`
   width: calc(var(--space-4) + var(--space-2));
   vertical-align: middle;
   overflow: hidden;
+  opacity: ${({ disabled }) => (disabled ? 0.5 : 1)};
   &:focus-within {
-    box-shadow 0 0 4px blue;
+    box-shadow: 0 0 4px blue;
   }
 `;
 
-export const Toggle = ({ value, onChange, label, ...props }) => (
-  <ToggleBody {...props}>
+export const Toggle = ({ value, onChange, label, disabled, trueLabel, falseLabel, ...props }) => (
+  <ToggleBody {...props} disabled={disabled}>
     <VisuallyHidden>
-      <input type="checkbox" checked={value} onChange={(e) => onChange(e.target.checked)} aria-label={label} />
+      <input type="checkbox" checked={value} onChange={(e) => onChange(e.target.checked)} aria-label={label} disabled={disabled} />
     </VisuallyHidden>
-    <Shuttle active={value} aria-hidden="true" />
+    <Shuttle active={value} aria-hidden="true" trueLabel={trueLabel} falseLabel={falseLabel} />
   </ToggleBody>
 );
 Toggle.propTypes = {
   value: PropTypes.any.isRequired,
   onChange: PropTypes.func.isRequired,
   label: PropTypes.string.isRequired,
+  disabled: PropTypes.bool,
+  trueLabel: PropTypes.string,
+  falseLabel: PropTypes.string,
+};
+Toggle.defaultProps = {
+  disabled: false,
+  trueLabel: '',
```

Comment:
Should we have something default here?

Other lgtm

Reply:
I think it makes sense for the default labels to be blank (as they are on iOS), but we can add labels if we want them